numpy/random/_generator.pyx

```diff
@@ -1535,6 +1535,8 @@ cdef class Generator:
         The corresponding probability density function for ``n = 20`` 
         and ``m = 20`` is:
         
+        >>> import matplotlib.pyplot as plt
+        >>> import scipy.stats as stats
```

Comment:
```suggestion
        >>> from scipy import stats
```

Reply:
Don't forget to add `# doctest: +SKIP` to these `scipy` imports. Same for #25530

Reply:
@rkern I added the skip in this line as well ``>>> from scipy.stats import binom  # doctest: +SKIP``but now there is an error

Reply:
All of the descendant lines need the `# doctest: +SKIP`, too; it works on a per-line (well, per-`>>>`) basis. This is one of the reasons we try not to use `scipy` in our docstring examples too much.